tests/data/function_trailing_comma.py

```diff
@@ -61,16 +62,19 @@ def f(
         "a": 1,
         "b": 2,
     }["a"]
-    if a == {
-        "a": 1,
-        "b": 2,
-        "c": 3,
-        "d": 4,
-        "e": 5,
-        "f": 6,
-        "g": 7,
-        "h": 8,
-    }["a"]:
+    if (
+        a
+        == {
+            "a": 1,
+            "b": 2,
+            "c": 3,
+            "d": 4,
+            "e": 5,
+            "f": 6,
+            "g": 7,
+            "h": 8,
+        }["a"]
+    ):
```

Comment:
It doesn't seem ideal to update a test expectation, so initially I'm considering this a workaround while reading up on some more of the background and context via #1619 and 586d24236e6b57bc3b5da85fdbe2563835021076.